netmap/netmap_contract.go

```diff
@@ -685,9 +703,9 @@ func setConfig(ctx storage.Context, key, val any) {
 }
 
 func cleanup(ctx storage.Context, epoch int) {
-	balanceContractAddr := storage.Get(ctx, balanceContractKey).(interop.Hash160)
```

Comment:
We can migrate these in `netmap` and don't register on balance/container update. But I don't see any clear advantage either way.

Reply:
> We can migrate these in netmap and don't register on balance/container update.

too hard to control it IMO. how to decide when a contract needs to be registered if you are a stupid contract? or rely on migration and never ever register container and balance again?

Reply:
Suppose you have updated netmap, but have not updated balance/container, what's gonna happen? In an ideal world, it all will work the same because netmap would migrate its callback thingies internally, not requiring old contracts to register somehow to continue their work.

Reply:
doen